packages/fxa-auth-server/lib/senders/templates/multiSubscriptionsPaymentExpired.html

```diff
@@ -0,0 +1,19 @@
+{{#*inline "title"}}
```

Comment:
Super-tiny nit that's more of a preference than anything: I was trying to prefix all these filenames and metric event things with "subscription..." (i.e. rather than something like "multi") so they naturally grouped together in alpha-sorted listings.